source/stylesheets/modules/_toc.scss

```diff
@@ -44,3 +44,7 @@
     display: none;
   }
 }
+
+.toc-link--in-view {
```

Comment:
That's fine, but we should probably then be using `.toc-link` on every toc link…